README.md

```diff
@@ -40,15 +42,53 @@ If you already have Rust installed, make sure you're using the latest version by
 rustup update
 ```
 
-Once done, finish installing the support software:
+Once done, finish installing the support software and configure your default toolchain:
 
 ```bash
-# Additional rust targets
-rustup target add wasm32-unknown-unknown --toolchain nightly-2020-10-18
+# Install nightly toolchain and use it
+rustup toolchain install nightly-2022-10-18
+rustup default nightly-2022-10-18
+rustup target add --toolchain nightly-2022-10-18 wasm32-unknown-unknown
```

Comment:
I think @JGiter @knzeng-e and I found that we needed `nightly-2022-09-22` (rust v1.66) to get compilation to work. However, I think we can leave as is and adjust in another PR as necessary.